src/main/java/com/github/fishio/control/SinglePlayerController.java

```diff
@@ -258,9 +258,6 @@ public void revive() {
 		ICollisionArea area = pf.getStartCollisionArea();
 		player.setBoundingArea(area);
 		
-		//Reset the speed of the fish.
-		player.setSpeedVector(new Vec2d(0, 0));
-		
```

Comment:
However, I think our restart and revive method in SinglePlayerController aren't doing it the proper way. (Restart should just completely remove the current playingField and revive should restart, but save some settings before restarting and setting them after it has started again). But that is something for the future.

<bug